### DISCUSSION TITLE
Labels, RGB Icons, Discord RPC (again)

### DISCUSSION THREAD

**Discussion by prevter** [Prevter/GDOpenHack#99] [Announcements]
**New**:
* Reimplemented "Discord RPC" with x4 more features
* Added "Click Teleport"
* Added "Uncomplete Level"
* Added "Labels"
* Added "Noclip Limit"
* Added "RGB Icons"
* Re-added search bar

**Fixes**:
* Fixed tooltips going off the screen
* Fixed saving settings for non-English languages
* Disabled "Speedhack" on game launch to prevent softlocks
* Various bug fixes
* *[Vanilla]* Fixed label colors for "StartPos Switcher"

<hr /><em>This discussion was created from the release <a href='https://github.com/Prevter/GDOpenHack/releases/tag/2.1.0'>Labels, RGB Icons, Discord RPC (again)</a>.</em>

**Comment by nfzchallengelist** [comment]
why is there no auto deafen yettttt ejkrgnalsdjkhASJKLDF

**Comment by ghost** [comment]
Labels are very op, but need update with more placeholders like **_gd currently using version,_** also need **_hitbox update_** because this have almost all gd menu have this feature. I think all want **_Hitbox_** , _**Layout mode**_  _**and texture pack**_ loader update.

**Reply by Askar4iklol** [reply]
i think that layout mode should be bc hitboxes will be added and geode version users can add texture loader mod
